core/trino-spi/pom.xml

```diff
@@ -254,28 +254,6 @@
                                     <new>method void io.airlift.slice.SliceOutput::write(byte[])</new>
                                     <exception>java.io.IOException</exception>
                                 </item>
-                                <item>
```

Comment:
above (`<old>method void io.airlift.slice.SliceOutput::write(byte[]) throws java.io.IOException</old>`) can also be removed